privacyidea/webui/login.py

```diff
@@ -44,10 +45,22 @@
 from privacyidea.lib.queue import has_job_queue
 
 DEFAULT_THEME = "/static/contrib/css/bootstrap-theme.css"
+DEFAULT_LANGUAGE_LIST = ['en', 'de', 'nl', 'zh_Hant', 'fr', 'es', 'tr']
 
 login_blueprint = Blueprint('login_blueprint', __name__)
 
 
+def get_accepted_language(req):
+    # if we are not in the request context, return None to use the default
+    # locale
+    if not req:
+        return None
+    # otherwise try to guess the language from the user accept
+    # header the browser transmits. The best match wins.
+    pi_cfg_configuration = get_app_config_value("PREFERRED_LANGUAGE", default=DEFAULT_LANGUAGE_LIST)
+    # reads pi.cfg and checks if preferred language is set. Otherwise default list is selected.
```

Comment:
i think we need to switch the comments here with L58-59
Also the variable name is misleading, just use something like `pi_lang_list`

Reply:
fixed in f1975862ca4f27f3f4a5882c97848bc68a575245.